chatcommands.py

```diff
@@ -1411,7 +1411,9 @@ def command_report_post(ev_room, ev_user_id, wrap2, message_parts, message_url,
                 (post_data.post_id, post_data.site)):
             # Don't re-report if the post wasn't marked as a false positive. If it was marked as a false positive,
             # this re-report might be attempting to correct that/fix a mistake/etc.
-            output.append("Post {}: Already recently reported".format(index))
+            se_link = parsing.to_protocol_relative(post_data.post_url)
+            ms_link = "https://m.erwaysoftware.com/posts/by-url?url={}".format(se_link)
+            output.append("Post {}: Already recently reported [[MS]({})]".format(index, ms_link))
```

Comment:
Could you add a check to see whether `GlobalVars.metasmoke_key` exists before appending the MS link? No point adding it unless if there is actually a report on MS